MusicTaste/MusicTaste/View Controllers/DetailsViewController.m

```diff
@@ -61,25 +63,28 @@ - (IBAction)panAction:(id)sender {
     }
     else if (self.panGesture.state == UIGestureRecognizerStateEnded) {
         if (self.isAcceptReached) {
-            [self.navigationController popToRootViewControllerAnimated:YES];
+            //Removes user from matches array and segues to MatchmakingViewController
```

Comment:
Once the accept or rejected is reached then we will remove the user from the matches array and segue back to the MatchmkaingViewController